go/mysql/flavor_mariadb.go

```diff
@@ -154,6 +155,25 @@ func (mariadbFlavor) status(c *Conn) (SlaveStatus, error) {
 	if err != nil {
 		return SlaveStatus{}, vterrors.Wrapf(err, "SlaveStatus can't parse MariaDB GTID (Gtid_Slave_Pos: %#v)", resultMap["Gtid_Slave_Pos"])
 	}
+	filePos, err := strconv.Atoi(resultMap["Exec_Master_Log_Pos"])
+	if err != nil {
+		return SlaveStatus{}, fmt.Errorf("invalid FilePos GTID (%v): expecting pos to be an integer", resultMap["Exec_Master_Log_Pos"])
+	}
+
+	status.FilePosition.GTIDSet = filePosGTID{
+		file: resultMap["Relay_Master_Log_File"],
```

Comment:
We should also check whether this value is filled in before setting a non-zero value for the position field.

Reply:
I was copying the pattern from the file position flavor, but good idea. Will do.

Reply:
What about the position field. Should we only set the field if both file and position retrieved are non-zero values?

Reply:
A literal "0" for the pos is ok, but we should leave the position empty if the pos is an empty string "".

Reply:
Done.